server/integration_tests/explore_test.py

```diff
@@ -120,12 +120,22 @@ def handle_response(self,
     if 'variables' in resp:
       resp_var_to_score = {}
       for i, sv in enumerate(dbg['sv_matching']['SV']):
-        score = dbg['sv_matching']['CosineScore'][i]
-        resp_var_to_score[sv] = float("{:.6f}".format(score))
+        truncated_score = _format_score(dbg['sv_matching']['CosineScore'][i])
+        resp_var_to_score[sv] = truncated_score
+        dbg['sv_matching']['CosineScore'][i] = truncated_score
```

Comment:
maybe we can move this part outside of the for loop & just have `dbg['sv_matching']['CosineScore'] = _format_scores(dbg['sv_matching'].get('CosineScore', [])`

& that way we just need one of the two helper functions & can read score from this formatted list

Reply:
done!